core/arch/arm/plat-stm32mp1/scmi_server.c

```diff
@@ -795,6 +794,21 @@ static int32_t pmic_set_state(struct stm32_scmi_voltd *voltd, bool enable)
 	return rc ? SCMI_GENERIC_ERROR : SCMI_SUCCESS;
 }
 
+static int32_t stub_describe_levels(struct stm32_scmi_voltd *voltd __unused,
+				    size_t start_index, long *microvolt,
+				    size_t *nb_elts)
+{
+	if (!microvolt || !*nb_elts || start_index) {
+		*nb_elts = 1;
+		return SCMI_SUCCESS;
+	}
+
+	microvolt[0] = 0;
```

Comment:
can check the IO level  => need to return the expected value else unexpected driver behavior.

if (!strcmp(voltd->priv_id, PWR_VREFBUF_NAME_ID)) {
    microvolt[0] = 0; /* VREFBUF not supported */
} else {
    microvolt[0] = 3300000; /* SDMMC IO = fixed at 3.3V, HighSpeed support */
}